pkg/kubelet/cm/topologymanager/socketmask/socketmask.go

```diff
@@ -0,0 +1,154 @@
+package socketmask
+
+import (
+	"strconv"
+        "strings"
+        "bytes"
+        "math"
+        "github.com/golang/glog"
+)
+
+
+type SocketMask []int64
+
+func NewSocketMask(Mask []int64) SocketMask {
+	sm := Mask
+	return sm
+}
+
+func (sm SocketMask) GetSocketMask(socketMask []SocketMask, maskHolder []string, count int) (SocketMask, []string) {
+	var socketAffinityInt64 [][]int64
+      	for r := range socketMask {
+       		socketAffinityVals := []int64(socketMask[r])
+          	socketAffinityInt64 = append(socketAffinityInt64,socketAffinityVals)
+    	}
+       	if count == 0 {
+            	maskHolder = buildMaskHolder(socketAffinityInt64)
+	}
+      	glog.Infof("[socketmask] MaskHolder : %v", maskHolder) 
```

Comment:
please change `glog` with `klog` from `k8s.io/klog`.
also ideally `Infof()` calls should be with verbosity level `V(x)`.